xbmc/video/videosync/VideoSyncDRM.cpp

```diff
@@ -151,18 +149,9 @@ void CVideoSyncDRM::EventHandler(int fd, unsigned int frame, unsigned int sec,
 {
   drmVBlank vbl;
   VblInfo *info = (VblInfo*)data;
-  int crtc = g_Windowing.GetCrtc();
+  drmVBlankSeqType crtcSel = CrtcSel();
```

Comment:
boot time was longer today :) I did this because the window can be moved from one monitor to the other.
all fine, thanks.